include/server/Settings.hpp

```diff
@@ -15,27 +15,27 @@ class Settings
   struct LobbySettings
   {
     // Bind address and port of the lobby host (default values)
-    std::string address = "localhost";
+    std::string address = "127.0.0.1";
```

Comment:
I think I initially had it like this, but regent asked to use localhost instead. Should work both ways, just not sure which one he preferred or why
I assume the resolve address function won't break with the address already resolved? I can't test this today but if it works then it's good

Reply:
The problem is that the resolving is done during parsing, while these default values skip the parsing, so it never gets resolved to 127.0.0.1
A value of localhost in the json file does work, but in these default values it doesnt get translated to 127.0.0.1

Reply:
Makes sense, I didn't think about that